### PR TITLE
WFIRST Cycle 6 / MCR data paths and version update

### PR DESCRIPTION
**Pull request by josePhoenix**
The majority of these changes are in the reference data (as they should be!) but this PR is to bump the minimum data version and change some paths to files whose names have changed.

### CODE REVIEW

File: webbpsf/wfirst.py [hunk 1]
```diff
@@ -299,14 +299,14 @@ def __init__(self):
...
-        self._detectors = _load_wfi_detector_aberrations(os.path.join(self._datapath, 'zernikes.csv'))
+        self._detectors = _load_wfi_detector_aberrations(os.path.join(self._datapath, 'wim_zernikes_cycle6.csv'))
         assert len(self._detectors.keys()) > 0
         self.detector = 'SCA01'
 
         # Paths to the two possible pupils. The correct one is selected based on requested
         # wavelengths in _validateConfig()
-        self._unmasked_pupil_path = os.path.join(self._WebbPSF_basepath, 'AFTA_WFC_C5_Pupil_Shortwave_Norm_2048px.fits')
-        self._masked_pupil_path = os.path.join(self._WebbPSF_basepath, 'AFTA_WFC_C5_Pupil_Mask_Norm_2048px.fits')
+        self._unmasked_pupil_path = os.path.join(self._WebbPSF_basepath, 'WFIRSTMCRWFCPupilShortwaveNorm2048px.fits')
```
Comment (mperrin) [inline webbpsf/wfirst.py]: These filenames are kind of terrible. I assume they are just direct copies of Goddard provide filenames? Does it make sense to rename to something cleaner? Even just adding back in the underscores would make it more readable.

**Comment by mperrin** [comment]
Got it. This needs the updated data files for any of the tests to run - are you going to make that or should I? If the contents of the data source directory are ready to go now, just let me know.

File: webbpsf/wfirst.py (hunk 1 above)
Comment (josePhoenix) [inline webbpsf/wfirst.py]: Yeah, we can name them whatever we want. I want to at least preserve their provenance (cycle 5 or MCR or whatever) in the filename so they can coexist with the old stuff in the data package during development and so newer versions can't accidentally use old files.

**Comment by josePhoenix** [comment]
I think they're ready, but I just changed some filenames so I want to run the tests again locally and verify they pass. Will comment here when it's ready for you.

**Comment by josePhoenix** [comment]
Tests passing locally! (Remember to copy in the data sources and add a `0.5.0` version.txt.)

**Pull request merged**